File: Examples/CSharp/SMTP/LoadSmtpConfigFile.cs
using System;
using System.Configuration;
using System.Diagnostics;
using Aspose.Email.Mime;
using Aspose.Email.Clients.Smtp;
using Aspose.Email.Clients;

namespace Aspose.Email.Examples.CSharp.Email.IMAP
{
    class LoadSmtpConfigFile
    {
        public static void Run()
        {
            // The path to the File directory.
            string dataDir = RunExamples.GetDataDir_SMTP();
            string dstEmail = dataDir + "EmbeddedImage.msg";

            // Declare msg as MailMessage instance
            MailMessage msg = new MailMessage();

            // Use MailMessage properties like specify sender, recipient and message
            msg.To = "asposetest123@gmail.com";
            msg.From = "aspose2@gmail.com";
            msg.Subject = "Test Email";
            msg.Body = "Hello World!";

            // ExStart:LoadSmtpConfigFile
            // Create an instance of SmtpClient class and load SMTP Authentication settings from Config file
            SmtpClient client = new SmtpClient(ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.None));
            // ExEnd:LoadSmtpConfigFile

            client.SecurityOptions = SecurityOptions.Auto;

            try
            {
                // Client.Send will send this message
                client.Send(msg);
                Console.WriteLine("Message sent");
            }
            catch (Exception ex)
            {
                Trace.WriteLine(ex.ToString());
            }
            Console.WriteLine(Environment.NewLine + "Message sent after loading configuration from config file.");
        }
    }
}

File: Examples/VisualBasic/SMTP/LoadSmtpConfigFile.vb
Imports System.IO
Imports Aspose.Email.Mail
Imports Aspose.Email.Outlook
Imports Aspose.Email.Pop3
Imports Aspose.Email
Imports Aspose.Email.Mime
Imports Aspose.Email.Imap
Imports System.Configuration
Imports System.Data
Imports Aspose.Email.Mail.Bounce
Imports Aspose.Email.Exchange
Imports Aspose.Email.Outlook.Pst

Namespace Aspose.Email.Examples.VisualBasic.Email.SMTP

    Public Class LoadSmtpConfigFile
        Public Shared Sub Run()
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_SMTP()
            Dim dstEmail As String = dataDir & Convert.ToString("EmbeddedImage.msg")

            'Declare msg as MailMessage instance
            Dim msg As New MailMessage()

            'use MailMessage properties like specify sender, recipient and message
            msg.[To] = "asposetest123@gmail.com"
            msg.From = "aspose2@gmail.com"
            msg.Subject = "Test Email"
            msg.Body = "Hello World!"

            'Create an instance of SmtpClient class and load SMTP Authentication settings from Config file
            Dim client As New SmtpClient(ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.None))
            client.SecurityOptions = SecurityOptions.Auto

            Try
                'Client.Send will send this message
                client.Send(msg)
                'Message sent successfully
                System.Console.WriteLine("Message sent")

            Catch ex As System.Exception
                System.Diagnostics.Trace.WriteLine(ex.ToString())
            End Try

            Console.WriteLine(Environment.NewLine + "Message sent after loading configuration from config file.")
        End Sub
    End Class
End Namespace